name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/merkurprivatbank/MerkurPrivatBankPDFExtractorTest.java

```diff
@@ -0,0 +1,90 @@
+package name.abuchen.portfolio.datatransfer.pdf.merkurprivatbank;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsEmptyCollection.empty;
+import static org.junit.Assert.assertNull;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import name.abuchen.portfolio.datatransfer.Extractor.BuySellEntryItem;
+import name.abuchen.portfolio.datatransfer.Extractor.Item;
+import name.abuchen.portfolio.datatransfer.Extractor.SecurityItem;
+import name.abuchen.portfolio.datatransfer.actions.AssertImportActions;
+import name.abuchen.portfolio.datatransfer.pdf.MerkurPrivatBankPDFExtractor;
+import name.abuchen.portfolio.datatransfer.pdf.PDFInputFile;
+import name.abuchen.portfolio.model.BuySellEntry;
+import name.abuchen.portfolio.model.Client;
+import name.abuchen.portfolio.model.PortfolioTransaction;
+import name.abuchen.portfolio.model.Security;
+import name.abuchen.portfolio.model.Transaction.Unit;
+import name.abuchen.portfolio.money.CurrencyUnit;
+import name.abuchen.portfolio.money.Money;
+import name.abuchen.portfolio.money.Values;
+
+@SuppressWarnings("nls")
+public class MerkurPrivatBankPDFExtractorTest
+{
+    @Test
+    public void testWertpapierKauf01()
+    {
+        MerkurPrivatBankPDFExtractor extractor = new MerkurPrivatBankPDFExtractor(new Client());
+
+        List<Exception> errors = new ArrayList<>();
+        List<Item> results = extractor.extract(PDFInputFile.loadTestCase(getClass(), "Kauf01.txt"), errors);
+
+        assertThat(errors, empty());
+        assertThat(extractor.getLabel(), is("MERKUR PRIVATBANK KGaA"));
+        assertThat(results.size(), is(2));
+        
+        new AssertImportActions().check(results, CurrencyUnit.EUR);
+
+        // Security identification:
+        // Stück 125,3258 XTR.(IE) - MSCI WORLD IE00BJ0KDQ92
+        // (A1XB5U)
+        // REGISTERED SHARES 1C O.N.
+        Security security = results.stream().filter(SecurityItem.class::isInstance).findFirst()
+                        .orElseThrow(IllegalArgumentException::new).getSecurity();
+        assertThat(security.getIsin(), is("IE00BJ0KDQ92"));
+        assertThat(security.getWkn(), is("A1XB5U"));
+        assertNull(security.getTickerSymbol());
+        assertThat(security.getName(), is("XTR.(IE) - MSCI WORLD REGISTERED SHARES 1C O.N."));
+        // assertThat(security.getCurrencyCode(), is(CurrencyUnit.EUR));
```

Comment:
remove `\\`

If you have implemented these changes in this comment --> https://github.com/buchen/portfolio/pull/3325/files#r1186609830 <-- then this test will run successfully.

Here we set the currency of the security!

In line 44 we check the ImportAction and thus also whether the security was created in a currency other than EUR. (foreign currency calculation etc.)